### PR TITLE
Fix: Basic Gallery - SVG Icon is not displayed in the Basic Gallery widget [ED-4046]

### PR DESCRIPTION
**Pull request by guyc-E**
…idget [ED-4046]

## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

[ED-4046]: https://elementor.atlassian.net/browse/ED-4046

### CODE REVIEW

File: assets/dev/js/editor/controls/gallery.js [hunk 1]
```diff
@@ -93,6 +102,19 @@ ControlMediaItemView = ControlBaseDataView.extend( {
 		}, this );
 	},
 
+	setUploadMimeType() {
```
Comment (matipojo) [inline assets/dev/js/editor/controls/gallery.js:105]:
```suggestion
	addSvgMimeType() {
	if ( ! FilesUploadHandler.isUploadEnabled( 'svg' ) ) {
		return;
	}
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by kobizz — CHANGES_REQUESTED** [review]

**Comment by kobizz** [comment]
@guyc-E Please merge release/3.4.0

File: assets/dev/js/editor/controls/gallery.js [hunk 2]
```diff
@@ -93,6 +102,19 @@ ControlMediaItemView = ControlBaseDataView.extend( {
...
+		// Set svg as only allowed upload extensions
```
[the remark this answers is not in the document]
Comment (guyc-E) [reply assets/dev/js/editor/controls/gallery.js:106]: You're right. I copied this code from another file and changed it but forgot to change the function's name and the comment.

File: assets/dev/js/editor/controls/gallery.js (hunk 1 above)
Comment (guyc-E) [reply assets/dev/js/editor/controls/gallery.js:105]: Thanks

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**